Godot XR Community update March 2025 - Godot XR Game Jam III (#1042)
godotengine/godot-website: collections/_article/godot-xr-update-mar-2025.md

---
title: "Godot XR update - March 2025"

> excerpt: "Godot XR Community Game Jam February 2024"

Reviewer: Don't forget to change excerpt!

categories: ["progress-report"]
author: Bastiaan Olij
image: /storage/blog/covers/march-2025-update-godot-xr-community.webp
image_caption_description: "Godot XR Gamejam III logo"
date: 2025-03-31 12:00:00
---

> In the second half of February 2025 the [Godot XR Community](https://godot-xr.itch.io/) held their latest [Godot XR gamejam](https://itch.io/jam/godot-xr-game-jam-feb-2025).

Reviewer: ```suggestion
In the second half of February 2025, the [Godot XR Community](https://godot-xr.itch.io/) held their latest [Godot XR Game Jam](https://itch.io/jam/godot-xr-game-jam-feb-2025).
```

> It is organised by [Bastiaan Olij](https://www.youtube.com/BastiaanOlij) and [Malcolm Nixon](https://www.youtube.com/@MalcolmANixon) acting in their role as representatives of the general Godot XR Community.

Reviewer: ```suggestion
It was organised by [Bastiaan Olij](https://www.youtube.com/@BastiaanOlij) and [Malcolm Nixon](https://www.youtube.com/@MalcolmANixon) acting in their roles as representatives of the general Godot XR Community.
```

> This was a week long event in which developers had to create an XR application using the theme "Enchanted". VR has great potential on this topic, whether you interpret this as physical spell casting, a magical environment, or something else and the entries did not disappoint.

Reviewer: ```suggestion
This was a week-long event in which developers had to create an XR application using the theme "Enchanted". XR has great potential on this topic, whether you interpret this as physical spell casting, a magical environment, or something else, and the entries did not disappoint.
```

> Compared to our previous gamejam we went from 26 entries and 283 ratings to 27 entries and 290 ratings. Not a great jump however we had 77 people join our previous game jam compared to 125 this jam.

Reviewer: ```suggestion
Compared to our previous game jam, we went from 26 entries and 283 ratings to 27 entries and 290 ratings. Not a great jump, however we had 125 people join this game jam compared to 17 in our previous jam.
```

Author: 77, not 17 :)

Reviewer: Oops!

> Based on feedback we've received on our Discord channel the main reason the higher participation did not translate to more entries was due to a number of participants not finishing their entry in time. We've had overwhelmingly positive feedback on the experience with many new developers becoming active members in our XR community.

Reviewer: ```suggestion
Based on feedback we've received on our Discord channel, the main reason the higher participation did not translate to more entries was due to a number of participants not finishing their entry in time. We've had overwhelmingly positive feedback on the experience with many new developers becoming active members in our XR community.
```

> While subjective, the overal quality of the entries seems to have been a lot higher than the previous jam so we are really looking forward to what the future will bring.

Reviewer 2: ```suggestion
While subjective, the overall quality of the entries seems to have been a lot higher than the previous jam so we are really looking forward to what the future will bring.
```

## Top 5

> So, let's take a closer look at the top 5 entries.

Reviewer 2: Very subjective, but I feel like this should be exclaimed!

```suggestion
So, let's take a closer look at the top 5 entries!
```

Author: ROFL, though @JohnVeness would comment on the exclamation mark, thumbsdown it is:)

I have no opinion, but I don't think we should rule out exclamation marks by default, sometimes they do add a bit extra. What do others think in this situation?

Reviewer 3: I would prefer an exclamation mark here and would also remove "So".

```suggestion
Let's take a closer look at the top 5 entries!
```

Author: Well that's 3 for !, one against :P

Reviewer: Ehhh, I'm fine with it :)

### No 5. Syneed

[Syneed](https://itch.io/jam/godot-xr-game-jam-feb-2025/rate/3332396) by [Hiina](https://hiina.itch.io/) ranked 5th with 13 ratings (Score: 3.705).

> Our fifth place entry is a puzzle game where you need to manipulate a stream of particle using attractors.

Reviewer: ```suggestion
Our fifth place entry is a puzzle game where you need to manipulate a stream of particles using attractors.
```

> By adjusting these within the world the trajectory of the particles change and you can guide them past a number of stars.

Reviewer: ```suggestion
By adjusting these within the world, the trajectory of the particles changes and you can guide them past a number of stars.
```

The goal is for the particle stream to touch all stars.

<iframe width="560" height="315" src="https://www.youtube.com/embed/uwsC-RsR9dM" title="syneed: VR puzzle game." frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

### No 4. Keziah's House

[Keziah's House](https://itch.io/jam/godot-xr-game-jam-feb-2025/rate/3331240) by [Warping Realities VR](https://warpingrealities.itch.io/) ranked 4th with 12 ratings (Score: 3.764).

Our fourth place goes to this great escape room game.
You find yourself waking up trapped in a derelict old house.
Solve puzzles while casting magic spells to find your way out.

> **Need to record a video for this, none was provided by the author**

Reviewer 2: They did make a video:

https://www.youtube.com/watch?v=EK75Raq0i7c

I needed it to make it through one of the puzzles :-)

Author: Cool, I'll use that :)

### No 3. The Chant of Cthulhu

[The Chant of Cthulhu](https://itch.io/jam/godot-xr-game-jam-feb-2025/rate/3322870) by [Copper Tunic](https://copper-tunic.itch.io/) ranked 3rd with 12 ratings (Score: 3.833).

Our third place is a horror karaoke game.
You need to sing out loud the chants given and as you do so, more and more "clones" are created that repeat your words.

> As everything reaches a cresendo you summon the monsters of the deep.

Reviewer: ```suggestion
As everything reaches a crescendo, you summon the monsters of the deep.
```

All within a church environment that fits the concept well.
Very good use of recording and sound and with a freeplay mode you can extend the fun by manually placing clones that sing out your recorded song.

<iframe width="560" height="315" src="https://www.youtube.com/embed/7Yxv9bE2sIQ" title="The Chant of Cthulhu" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

### No 2. EZ Enchants

[EZ Enchants](https://itch.io/jam/godot-xr-game-jam-feb-2025/rate/3316696) by [Dragon1Freak](https://dragon1freak.itch.io/) ranked 2nd with 12 ratings (Score: 3.931).

> Our runner up is a fun little crafting game.

Reviewer: ```suggestion
Our runner-up is a fun little crafting game.
```

As customers walk past your little magic store, they ask you to enchant different objects.

> You have to perform different actions according to the customers wishes, all within a time limit.

Reviewer: ```suggestion
You have to perform different actions according to the customer's wishes, all within a time limit.
```

A very polished, good looking entry that is fun to play!

<iframe width="560" height="315" src="https://www.youtube.com/embed/ni4SI-HelwM" title="EZ Enchants - Full Run" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

### No 1. Your Hand Is A Dragon (XR)

[Your Hand Is A Dragon (XR)](https://itch.io/jam/godot-xr-game-jam-feb-2025/rate/3332432) by [Andyman404](https://andyman404.itch.io/) ranked 1st with 10 ratings (Score: 4.020).

> Our winning entry is a very unique game as we've come to expect from Andyman404.

Reviewer: ```suggestion
Our winning entry is a very distinct game, as we've come to expect from Andyman404.
```
Or similar. Something can be unique or not, there aren't graduations!

You play as a dragon that attacks fortresses manned by knights.
What makes it unique is that you control the dragon with your hand as if it is a sockpuppet,

> while simultaniously roaring out loud to make the dragon breath fire.

Reviewer: ```suggestion
while simultaneously roaring out loud to make the dragon breathe fire.
```

It's especially impressive as Andyman404 only had 3 days to dedicate to this entry.

<iframe width="560" height="315" src="https://www.youtube.com/embed/xP-1_EOkLZA" title="Your Hand Is A Dragon - Trailer" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>

## The next Godot XR Community game jam

A date has not been set for the next jam but it will likely be in August or September of this year.

> Communication around the game jam has moved to the newly created [Godot XR Community itch.io page](https://godot-xr.itch.io/) so be sure to follow that account.

Reviewer: ```suggestion
Communication around the game jam has moved to the newly created [Godot XR Community itch.io page](https://godot-xr.itch.io/), so be sure to follow that account.
```

> Join the Godot XR Community in the XR channel on [Godots official Discord server](https://discord.gg/godotengine) if you're interested in using Godot to develop XR projects. 

Reviewer: ```suggestion
Join the Godot XR Community in the XR channel on [Godot's official Discord server](https://discord.gg/godotengine) if you're interested in using Godot to develop XR projects. 
```

while simultaneously roaring out loud to make the dragon breathe fire.
It's especially impressive as Andyman404 only had 3 days to dedicate to this entry.

<iframe width="560" height="315" style="width: 100%; height: 100%; aspect-ratio: 16/9;" src="https://www.youtube.com/embed/xP-1_EOkLZA" title="Your Hand Is A Dragon - Trailer" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 
## Other entries

Malcolm Nixon created the below video on his YouTube channel that shows off the top 5 games and highlights many of the other entries.
It is well worth a watch to get an impression of the entries that were submitted.

<iframe width="560" height="315" style="width: 100%; height: 100%; aspect-ratio: 16/9;" src="https://www.youtube.com/embed/t5U6dXm8WY4" title="Godot XR Game Jam 3 - Top 5 Countdown" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
 
## The next Godot XR Community game jam

> A date has not been set for the next jam but it will likely be in August or September of this year.

Reviewer 4: ```suggestion
The next jam will likely be in August or September of this year.
```
